src/backend/p3/p3api.js

```diff
@@ -0,0 +1,50 @@
+const fs=require("fs/promises");
+const express = require("express");
+//const cors = require("cors"); //if encoutering errors whne using from the front end, remove comment status //didn't require it so far
+
+const app= express()
+app.use(express.json());
+//app.use(express.cors());  //if encoutering errors whne using from the front end, remove comment status s //didn't require it so far
+
+if (require.main === module) {
+  app.listen(3000, () => console.log('p3 server is running'));
```

Comment:
Consider moving your listener to the bottom of the page. Just best practice as js reads top to bottom and you want the server to start listening last